File: Ruby/ITP1_02_D.rb
n = gets.split(" ").map(&:to_i)

if 0 <= n[2] - n[4] && n[2] + n[4] <= n[0] && 0 <= n[3] - n[4] && n[3] + n[4] <= n[1] then
  puts "Yes"
else
  puts "No"
end

File: Python3/ITP1_02_D.py
n = [int(i) for i in input().split()]

if 0 <= n[2] - n[4] and n[2] + n[4] <= n[0] and 0 <= n[3] - n[4] and n[3] + n[4] <= n[1]:
    print("Yes")
else:
    print("No")
